.github/workflows/ci.yml

```diff
@@ -0,0 +1,147 @@
+name: "CI"
+
+on:
+    pull_request:
+    push:
+        branches:
+            - 'master'
+
+env:
+    fail-fast: true
+    TZ: "Europe/Paris"
+
+jobs:
+    phpstan:
+        name: "PHPStan (${{ matrix.php-version }})"
+
+        runs-on: "ubuntu-latest"
+
+        strategy:
+            matrix:
+                php-version:
+                    - '7.3'
+
+        steps:
+            - name: "Checkout code"
+              uses: actions/checkout@v2.3.3
+
+            - name: "Install PHP with extensions"
+              uses: shivammathur/setup-php@2.7.0
+              with:
+                  coverage: "none"
+                  php-version: "${{ matrix.php-version }}"
+                  tools: composer:v2
+            -
+                name: "Composer install"
+                uses: "ramsey/composer-install@v1"
+                with:
+                    composer-options: "--no-scripts"
```

Comment:
imo there's no need for this action to be used, just use `composer install --no-scripts`. Unless you really want to keep the cache ?

Reply:
I don't have much of a clue about this (this is mostly copied from PHPBench which was contributed by @OskarStark ). But isn't using the cache beneficial here?

Reply:
Hum, as you commit the composer.lock it may make sense.

Reply:
Initially I translated Travis.yaml to github actions in phpbench and it was part of the Travis.yaml 😉

If it is working, let's keep it for now and maybe check afterwards in a stand-alone PR what will be the difference